src/maat/web/__init__.py

```diff
@@ -120,6 +121,15 @@ def _copy_traversable(traversable: Traversable, dest: Path):
             dest_child.write_bytes(child.read_bytes())
 
 
+def _write_robots_txt(output: Path):
+    """Write robots.txt file to block all web crawlers."""
+    robots_content = """User-agent: *
+Disallow: /
+"""
```

Comment:
could you put `robots.txt` file in the `templates` folder and just use `_copy_traversable` to copy it into output dir?